airflow/providers/cohere/operators/embedding.py

```diff
@@ -0,0 +1,70 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+from functools import cached_property
+from typing import TYPE_CHECKING, Any, Sequence
+
+from airflow.models import BaseOperator
+from airflow.providers.cohere.hooks.cohere import CohereHook
+
+if TYPE_CHECKING:
+    from airflow.utils.context import Context
+
+
+class CohereEmbeddingOperator(BaseOperator):
+    """Creates the embedding base by interacting with cohere hosted services.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:CohereEmbeddingOperator`
+
+    :param conn_id: Optional. The name of the Airflow connection to get connection
+        information for Cohere. Defaults to "cohere_default".
+    :param input_text: list of text items that need to be embedded. Only one of input_text or input_callable
+        should be provided.
```

Comment:
```suggestion
     :param input_text: list of text items that need to be embedded. Only one of input_text or input_callable
        should be provided.
    :param conn_id: Optional. The name of the Airflow connection to get connection
        information for Cohere. Defaults to "cohere_default".
```
We should keep the order same as init method param